tests/nativeapp/test_manager.py

```diff
@@ -822,7 +822,10 @@ def test_create_dev_app_create_new_with_additional_privileges(
 
 @mock.patch(NATIVEAPP_MANAGER_EXECUTE)
 @mock.patch(CLI_GET_CONNECTION)
-def test_create_dev_app_bad_comment(mock_conn, mock_execute, temp_dir, mock_cursor):
+@pytest.mark.parametrize("loose_files_magic_version", LOOSE_FILES_MAGIC_VERSIONS)
```

Comment:
I chose to parameterize just one test with both strings; I think this adequately covers things but open to different approaches.